.github/workflows/dev-tool-workflow.yml

```diff
@@ -32,6 +32,11 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
 
+      - name: Install go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.22.3'
+
```

Comment:
Looks like this workflow/job is used by many jobs in `dev-tool-test` but only the added ones needs this step. Wouldn't be a problem for other unrelated jobs, _e.g._, JS jobs to install Go? Would this have a performance impact?

Reply:
@acuarica Well, I could make a separate workflow but it would be nearly identical to `dev-tool-workflow`. I assumed that `dev-tool-workflow` is common to all jobs. I don't see any performance impact here for example: https://github.com/hashgraph/hedera-json-rpc-relay/actions/runs/10037422122/job/27827745323?pr=2679